### PR TITLE
Correct runtime end of maintenance dates

### PR DESCRIPTION
**Pull request by feluelle**

### CODE REVIEW

**Comment by feluelle** [comment]
I did not ask, but I think the dates were incorrect. PTAL.

File: astro/runtime-version-lifecycle-policy.md [hunk 1]
```diff
@@ -86,8 +86,8 @@ The following table contains the exact lifecycle for each published version of A
 | [4](runtime-release-notes.md#astro-runtime-420) | 2.2                    | March 10, 2022     | September 2023          |
 | [5](runtime-release-notes.md#astro-runtime-500) | 2.3                    | April 30, 2022     | October 2023            |
 | [6](runtime-release-notes.md#astro-runtime-600) | 2.4                    | September 19, 2022 | March 2024              |
-| [7](runtime-release-notes.md#astro-runtime-700) | 2.5                    | December 3, 2022   | June 2023               |
-| [8](runtime-release-notes.md#astro-runtime-800) | 2.6                    | April 30, 2023                | October 2023            |
+| [7](runtime-release-notes.md#astro-runtime-700) | 2.5                    | December 3, 2022   | June 2024               |
+| [8](runtime-release-notes.md#astro-runtime-800) | 2.6                    | April 30, 2023     | October 2024            |
```
Comment (jedcunningham) [inline astro/runtime-version-lifecycle-policy.md:90]:
```suggestion
| [8](runtime-release-notes.md#astro-runtime-800) | 2.6                    | April 30, 2023     | October 2023            |
```

This one was right. It's not an LTS release, it gets 6 months.

File: astro/runtime-version-lifecycle-policy.md [hunk 2]
```diff
@@ -86,8 +86,8 @@ The following table contains the exact lifecycle for each published version of A
 | [4](runtime-release-notes.md#astro-runtime-420) | 2.2                    | March 10, 2022     | September 2023          |
 | [5](runtime-release-notes.md#astro-runtime-500) | 2.3                    | April 30, 2022     | October 2023            |
 | [6](runtime-release-notes.md#astro-runtime-600) | 2.4                    | September 19, 2022 | March 2024              |
-| [7](runtime-release-notes.md#astro-runtime-700) | 2.5                    | December 3, 2022   | June 2023               |
-| [8](runtime-release-notes.md#astro-runtime-800) | 2.6                    | April 30, 2023                | October 2023            |
+| [7](runtime-release-notes.md#astro-runtime-700) | 2.5                    | December 3, 2022   | June 2024               |
```
Comment (jedcunningham) [inline astro/runtime-version-lifecycle-policy.md:89]:
```suggestion
| [7](runtime-release-notes.md#astro-runtime-700) | 2.5                    | December 3, 2022   | July 2023               |
```

This is the right date. This one also isn't LTS, but it gets 3 months after the next major (8) was released.

Comment (jedcunningham) [reply astro/runtime-version-lifecycle-policy.md:89]: Don't feel bad though, having LTS and non-LTS in the "stable" table with no differentiation, then having a separate table of just "lts" is pretty confusing imo 🤷‍♂️. 4/5/6 all just happen to be LTS.

**Review by jedcunningham — CHANGES_REQUESTED** [review]

**Review by jedcunningham — APPROVED** [review]
I took the liberty to accept my suggestions since Felix is OOO for a bit.

**Review by jwitz — APPROVED** [review]
LGTM @jedcunningham. For my reference, is the support timeline for stable 7 months now, or have we extended 7's timeline for another reason?

**Pull request merged**